src/ui/buttons/util.js

```diff
@@ -25,8 +24,9 @@ export function calculateMessagePosition(
   const showPoweredBy = calculateShowPoweredBy(layout, fundingSources);
 
   if (showPoweredBy && position === MESSAGE_POSITION.BOTTOM) {
-    throw new ValidationError(
-      "Message position must be 'top' when Debit and/or Credit Card button is present"
+    // eslint-disable-next-line no-console
+    console.warn(
+      "PayPal Button Message cannot be positioned at bottom when displaying the Debit or Credit Card button."
```

Comment:
I just copied the new content from the AC bullet.  
I'm inclined to suggest we stick with that content since product thoughtfully wrote it.  It does have two improvements:
1. The old language suggests a change to "top" is required if they had entered "bottom", and this new language explains why its position is different than requested but does not demand change.
2. And the Debit or Credit Card button's content reads "Debit or Credit Card" so that actually became more accurate.

Reply:
thank you for the context! I can agree with those points on that.